### PR TITLE
Query and update

### PR DESCRIPTION
**Pull request by Alusp**
## I implemented the Following in this PR:

- [ ] Add a column species of type string to your animals table. Modify your schema.sql file.
- [ ] Insert the following data:
- Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape.
- Animal: Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times.
- Animal: His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to escape 3 times.
- Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once.
- Animal: His name is Boarmon. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times.
- Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times.
- Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times.
- [ ] Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.

- [ ] Inside a transaction: Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
- [ ] Inside a transaction Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
- [ ] Commit the transaction.
- [ ] Inside a transaction:
- Delete all animals born after Jan 1st, 2022.
- Create a savepoint for the transaction.
- Update all animals' weight to be their weight multiplied by -1.
- Rollback to the savepoint
- Update all animals' weights that are negative to be their weight multiplied by -1.
- Commit transaction

- [ ] Write queries to answer the following questions:

- How many animals are there?
- How many animals have never tried to escape?
- What is the average weight of animals?
- Who escapes the most, neutered or not neutered animals?
- What is the minimum and maximum weight of each type of animal?
- What is the average number of escape attempts per animal type of those born between 1990 and 2000?

01 unspecies-transaction
02 Digimon
03 digimon
04 Negative Transact
q1
q2
Real Transaction

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -8,3 +8,16 @@ INSERT INTO animals(id, name, date_of_birth, escape_attempts,neutered,weight_kg)
 
 INSERT INTO animals(id, name, date_of_birth, escape_attempts,neutered,weight_kg) VALUES(4, 'Devimon', '05-12-2017', 5, TRUE, 11);
 
+INSERT INTO animals(id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(5,'Charmander','02-08-2020',0,FALSE,11.0);
+
+INSERT INTO animals(id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(6,'Plantmon','11-15-2021',2,TRUE,5.7);
+
+INSERT INTO animals(id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(7,'Squirtle','04-02-1993',3,TRUE,12.13);
+
+INSERT INTO animals(id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(8,'Angemon','06-12-2005',1,FALSE,45);
```
Comment (t-yanick) [inline data.sql:17]: - [OPTIONAL] According to the project requirements, these values for weight are negative values, please kindly fix them.

File: queries.sql [hunk 2]
```diff
@@ -22,4 +22,88 @@ SELECT * FROM animals
...
+/* What is the minimum and maximum weight of each type of animal? */
+SELECT MIN(weight_kg) FROM animals
+vet_clinic-# GROUP BY species; 
+
+SELECT MAX(weight_kg) FROM animals
+vet_clinic-# GROUP BY species;
+
+/* What is the average number of escape attempts per animal type of those born between 1990 and 2000? */
+SELECT AVG(escape_attempts) FROM animals
+WHERE date_of_birth BETWEEN '01-01-1990' AND '01-01-2000'
```
Comment (t-yanick) [inline queries.sql:108]: - [OPTIONAL] I think here, the date_of_birth >= '1990/01/01' AND date_of_birth <= '2000/12/31', please kindly correct that.

**Review by t-yanick — APPROVED** [review]
Hi @Alusp, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alusp** [comment]
Thanks for the review @t-yanick

**Pull request merged**